### ISSUE TITLE
GUI problems on Ubuntu

### ISSUE THREAD

**Issue by RedactedProfile** [redis/RedisDesktopManager#3377]
Quick note: love the software and it's potential. It works perfectly for Localhost work.  This is for **Ubuntu**, I cannot vouch for the other builds. 

But it only fetches the key list using SSH Tunnel, and that's the extent of the functionality. 

When I load up a connection that isn't localhost, using SSH tunnel, it successfully connects and gets the key list as it should. When I click on ANY key at all, be it 1 character long or a json dump from an API, it doesn't matter: it simply will not load. It hangs on a blank interface while the console output reports some weird timeout issues.  
When i right click on the connection and use the Console there, I can successfully use it to do any redis operations within milliseconds, so thankfully that works there. 

Eventually when it's had enough of waiting for a key or two to load up data, it just crashes. 

Gives me this:

![screenshot from 2015-07-07 09 17 57](https://cloud.githubusercontent.com/assets/1039911/8551173/ddabad2a-2489-11e5-9353-18c262a3f681.png)

and when I click "OK" on the window telling me to update RDM (which btw I have the latest.. fresh install as of 30 minutes ago) it gives me this:

![screenshot from 2015-07-07 09 18 08](https://cloud.githubusercontent.com/assets/1039911/8551204/0678fc4e-248a-11e5-8389-dbb98a150895.png)

so it also fails to report sending the error report. 

Basically the software craps the bed when using ssh tunnel for anything other than keylist. 

I've been successfully able to repro this 3 times in a row, though the exact time in which the crash happens, varies.  Sometimes I need to try and load two keys, sometimes it just freaks out when I try to do anything else while waiting for the first key to load (Such as open the window to create another connection, open the console, pretty much anything)

**Comment by uglide** [comment]
Hello @RedactedProfile,
Thanks for the feedback. As I can see you are using 0.7.6 version of RDM. This version contains a lot of bugs which were fixed in 0.7.9.X builds (aka 0.8.0 beta).

Please try 0.7.9.X builds. Thanks!

**Comment by RedactedProfile** [comment]
Hi @uglide thank you for the response 

I will try that, thank you.

**Comment by uglide** [comment]
@RedactedProfile
The fastest way is:

```
wget http://redisdesktop.com/builds/redis-desktop-manager_Ubuntu14.04_0.7.9.178_amd64.deb && sudo dpkg -i redis-desktop-manager_Ubuntu14.04_0.7.9.178_amd64.deb
```

**Comment by RedactedProfile** [comment]
@uglide 

It definitely works now, though there are some graphical glitches here and there, I will chalk those up to being in beta. 

For brevities sake though, the glitches im encountering are:
1. When the key's value forces a vertical scroll bar, the text goes bold
2. When I scroll, sometimes the text just.. doesn't show up. Looks like it's white text on a white background because I can still select it and copy+paste elsewhere. 
3. When I resize the window with a value open that has the characteristics of glitch 1, the window resizing is very slow and graphically buggy

Thank you for pointing me in the right direction!

**Comment by uglide** [comment]
@RedactedProfile I tested latest RDM builds on ubuntu and looks like UI glitches is Qt bug related to OpenGL. So nothing to do here in RDM. Add this issue to "Known issues" page in wiki.
